pom.xml

```diff
@@ -351,8 +360,8 @@
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-compiler-plugin</artifactId>
                         <configuration>
-                            <source>1.7</source>
-                            <target>1.7</target>
+                            <source>${jdk.version}</source>
```

Comment:
@2686747 We could remove all this part. Why should we have a special compiler configuration for qulice profile ?